clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -43,6 +43,76 @@
     "icon": "resources/logo.png",
     "main": "./out/extension.js",
     "contributes": {
+        "problemMatchers": [
+            {
+                "name": "enterprise-cobol",
+                "owner": "Cobol LS",
+                "fileLocation": [
+                    "absolute"
+                ],
+                "pattern": [
+                    {
+                        "regexp": "(.*):(.*?):(\\d+)\\s-(\\d+)\\s-\\s(.*?):\\s(.*)",
+                        "file": 1,
+                        "line": 2,
+                        "column": 3,
+                        "endColumn": 4,
+                        "severity": 5,
+                        "message": 6,
+                        "loop": true
+                    }
+                ],
+                "source": "Cobol compiler"
+            }
+        ],
+        "taskDefinitions": [
+            {
+                "type": "cobolCompile",
+                "required": [
+                    "jobCard",
+                    "presentation",
+                    "saveSysprint",
+                    "sysPrintLocation"
+                ],
+                "properties": {
+                    "jobCard": {
+                        "type": "string",
+                        "description": "Relative path from workspace specifying the file containing the JOB card."
+                    },
+                    "steplib": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        },
+                        "default": [],
+                        "description": "Dataset name where COBOL compiler (IGYCRCTL) resides."
+                    },
+                    "syslib": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        },
+                        "description": "Copybooks library concatenation."
+                    },
+                    "presentation": {
+                        "type": "object",
+                        "default": {
+                            "revealProblems": "onProblem"
+                        }
+                    },
+                    "saveSysprint": {
+                        "type": "boolean",
+                        "default": true,
+                        "description": "Do you want to save SYSPRINT?"
+                    },
+                    "sysPrintLocation": {
+                        "type": "string",
+                        "description": "Specifies the location to store SYSPRINT file. If empty, it defaults to <WORKSPACE_LOC>/SYSPRINT/<JOBID>",
```

Comment:
By default it would be better to use a gitignored subfolder of `.c4z` like we use for downloaded copybooks or for symbolic date retrieved from protsym in Debugger. What about `workspace/.c4z/listings`?